packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -138,9 +138,15 @@ export const messages = {
   force_lightning_interface_create_text: 'SFDX: Create Aura Interface',
   force_function_create_text: 'SFDX: Create Function',
   force_function_start_text: 'SFDX: Start Function',
+  force_function_start_warning_no_toml:
+    'No function.toml found. Create function.toml or intialize a new function folder using SFDX: Create Function.',
+  force_function_start_warning_not_in_function_folder:
+    'Please open a function file to run SFDX: Start Function',
   force_function_invoke_text: 'SFDX: Invoke Function',
   force_function_invoke_tooltip: 'Send Request',
   force_function_stop_text: 'SFDX: Stop Function',
+  force_function_stop_in_progress: 'Stopping Function',
+  force_function_stop_not_started: 'No Function is running locally',
```

Comment:
How does customer recover from No function is running locally? Is it SFDX: Start Function? Right-click on a Functions file and then select it?

Reply:
@xyc ^^

Reply:
Usually this is something that user doesn't need to recover from, since if their intention was to stop the function container, their goal is already completed here. And this is a mild informational message to indicate that. Similar message can be found here:
https://github.com/forcedotcom/salesforcedx-vscode/blob/e9e90b928028123342143518cac1c26188965eb3/packages/salesforcedx-vscode-lwc/src/messages/i18n.ts#L35-L36

Reply:
Ok, thanks.